src/components/container/SurveyTaskList/SurveyTaskList.tsx

```
@@ -10,24 +10,26 @@ import { ButtonVariant } from '../../presentational/Button/Button'
-	titleColor?: ColorDefinition
-	cardBackgroundColor?: ColorDefinition
-	cardStyle?: React.CSSProperties
-	buttonVariant?: ButtonVariant
-	buttonColor?: ColorDefinition
+	status: SurveyTaskStatus;
+	limit?: number;
+	sequential?: boolean;
+	title?: string;
+	surveys?: string[];
+	category?: string;
+	onDetailLinkClick?: Function;
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)